### DISCUSSION TITLE
How to know list of track?

### DISCUSSION THREAD

**Discussion by tiwari1amrit** [ant-media/Ant-Media-Server#5238] [General]
Hi there, 
Currently, I am integrating Ant Media SDK in iOS. Due to the lack of documents for implementing Video Conferencing in the iOS App, I am stuck on implementing feature of conferencing, I saw a [file](https://github.com/ant-media/WebRTC-iOS-SDK/blob/master/WebRTC-Sample-App/ConferenceViewController.swift). which is fully messy. 

I publish video for local as
```
        self.conferenceClient.setWebSocketServerUrl(url: url)
        self.conferenceClient.setLocalView(container: self.videoView, mode: .scaleAspectFill)
        self.conferenceClient.joinRoom(roomId: roomId)
```
But how can I know how many people are in that room? I saw a delegate method on `AntMediaClientDelegate`
 ```
   func trackAdded(track: RTCMediaStreamTrack, stream: [RTCMediaStream]) {

        print(stream)
    }
```
Can anyone please help me how to know how many people on that room as I have a roomId?

**Comment by lastpeony** [comment]
You can do that with sending a rest api call to
https://antmedia.io/rest/#/BroadcastRestService/getConferenceRoom
This will return you a room object which contains roomStreamList
I suppose size of roomStreamList should be equal to the number of people in the room

**Reply by tiwari1amrit** [reply]
Thank you for your response, 
from the API `LiveApp/rest/v2/broadcasts/conference-rooms/roomIdConstant` I got all track lists including non-active. 
How can I get` stream id` of active user in a room?

**Reply by tiwari1amrit** [reply]
@mekya do you have any idea on this one?

**Reply by mekya** [reply]
There is also a websocket message `getRoomInfo` and it returns the active streams/tracks in the room. It's being used under the hood.
[Check this out](https://github.com/ant-media/WebRTC-iOS-SDK/blob/master/WebRTCiOSSDK/api/ConferenceClient.swift#L123
)
 
Server responds with [ROOM_INFORMATION_COMMAND](https://github.com/ant-media/WebRTC-iOS-SDK/blob/master/WebRTCiOSSDK/api/ConferenceClient.swift#L133)

Sorry for late reply. Please check and let me know if I can help further
